docs/buildExternalDocs.bash

```diff
@@ -80,6 +80,7 @@ do
   [ -e "$file" ] || continue
   tail -n +3 "$file" > "$file.tmp" && mv "$file.tmp" "$file"
   sed -i -e 's/README/index/g' $file
+  sed -i -e 's,../modules/Constraint_eDSL.,../../constraints-edsl-api/modules/,g' $file
```

Comment:
Can this line be added to the processing done in `./source/scheduling-edsl-api/enums/` and `./source/scheduling-edsl-api/classes/`?